product/reports/670_Performance by Asset Type - Middleware Servers/160_Web Metrics - hourly over last day.yaml

```diff
@@ -0,0 +1,59 @@
+---
+where_clause:
+dims:
+reserved:
+title: "EAP Web Metrics - hourly over the last day"
+conditions:
+order: Descending
+graph:
+menu_name: "EAP Web Metrics - hourly over the last day"
+rpt_group: Custom
+priority:
+col_order:
+- id
+- name
+- start_time
+- mw_aggregated_active_web_sessions_max
+- mw_aggregated_max_active_web_sessions_max
+- mw_aggregated_expired_web_sessions_max
+- mw_aggregated_rejected_web_sessions_max
+- mw_aggregated_servlet_time_avg
+- mw_aggregated_servlet_request_count_max
```

Comment:
I see this is always zero in your screenshot. Probably more time was needed to let the agent make measurements.

Could you confirm that this isn't an always increasing number? If so, you will need to do some extra processing to show the true _hourly request count_ rather than an accumulated count.

Reply:
Now I understand that `mw_aggregated_servlet_time` and `mw_aggregated_servlet_request_count` are under `subsystem=servlet` that is why they are zeros.

Reply:
Regarding to the request count, it is not an absolute count. It only counts the request per hour. Not adding up the current count to the previous hour count.
Is that what you mean?

Reply:
Yes, my reasoning was that counters... well... they usually never reset to zero to let you query them and see the total count at the "present" time. So, I was expecting the request count to be an always increasing number. But from your updated image, I see that's not the case.